lib/dataContract/validateDataContractFactory.js

```diff
@@ -8,12 +9,17 @@ const InvalidIndexPropertyTypeError = require('../errors/InvalidIndexPropertyTyp
 const SystemPropertyIndexAlreadyPresentError = require('../errors/SystemPropertyIndexAlreadyPresentError');
 const DataContractRestrictedIdentityError = require('../errors/DataContractRestrictedIdentityError');
 const UniqueIndicesLimitReached = require('../errors/UniqueIndicesLimitReached');
+const DataContractMaxByteSizeExceededError = require('../errors/DataContractMaxByteSizeExceededError');
 
 const getPropertyDefinitionByPath = require('./getPropertyDefinitionByPath');
 
+const { encode } = require('../util/serializer');
+
 const systemProperties = ['$id', '$userId'];
 const prebuiltIndices = ['$id'];
 
+const MAX_SERIALIZED_DATA_CONTRACT_BYTE_SIZE = 15 * 1024; // 15 Kb
```

Comment:
Let's move this constant to `DataContractMaxByteSizeExceededError` as a class constant then we could reuse it here and in the test. We already doing this way.